colcon_sanitizer_reports/xml_output_generator.py

```diff
@@ -56,7 +56,11 @@ def _create_error_report(self, base_element: eTree.Element) -> eTree.Element:
         error_count_by_package = defaultdict(int)  # type: Dict[str, int]
         testcases = defaultdict()  # type: Dict[str, eTree.Element]
         for case in base_element.findall('testcase'):
-            testcases[case.get('name')] = case
+            case_name = case.get('name', None)
```

Comment:
```suggestion
            case_name = case.get('name')
```

None is already the default

Reply:
Is this a case of [`Explicit is better than implicit`](https://www.python.org/dev/peps/pep-0020/) 😅 ?

Reply:
Fair enough, LGTM